anta/inventory/__init__.py

```diff
@@ -342,3 +342,33 @@ async def connect_inventory(self) -> None:
             if isinstance(r, Exception):
                 message = "Error when refreshing inventory"
                 anta_log_exception(r, message, logger)
+
+    def dump(self, path: Path) -> None:
+        """Dump the AntaInventory to path.
+
+        Parameters
+        ----------
+        path
+            The Path to dump the inventory to
+        """
+        hosts = [
+            AntaInventoryHost(
+                name=device.name,
+                host=device.host if hasattr(device, "host") else device.name,
+                port=device.port if hasattr(device, "port") else None,
+                tags=device.tags,
+                disable_cache=device.cache is None,
+            )
+            for device in self.devices
+        ]
+
+        try:
+            with path.open("w", encoding="UTF-8") as file:
+                safe_dump(
+                    {AntaInventory.INVENTORY_ROOT_KEY: {"hosts": [safe_load(host.model_dump_json(serialize_as_any=True, exclude_unset=True)) for host in hosts]}},
```

Comment:
Maybe get rid of `INVENTORY_ROOT_KEY` and write the right pydantic model?

Reply:
2.0.0